pkg/sdn/plugin/ovscontroller.go

```diff
@@ -364,7 +364,17 @@ func (oc *ovsController) UpdatePod(sandboxID string, vnid uint32) error {
 	return oc.setupPodFlows(ofport, podIP, podMAC, note, vnid)
 }
 
-func (oc *ovsController) TearDownPod(hostVeth, podIP string) error {
+func (oc *ovsController) TearDownPod(hostVeth, podIP, sandboxID string) error {
+	if podIP == "" {
+		flows, err := oc.ovs.DumpFlows()
+		if err != nil {
+			return err
+		}
+		if _, ip, _, _, err := getPodDetailsBySandboxID(flows, sandboxID); err == nil {
+			podIP = ip
```

Comment:
We may get an error here if the corresponding ovs flow rule doesn't exists which means tearDown already happened. Don't we want to ignore in this case?